lib/api/objectGet.js

```diff
@@ -106,6 +111,11 @@ function objectGet(authInfo, request, returnTagCount, log, callback) {
             // objMD.location is just a string
             dataLocator = Array.isArray(objMD.location) ?
                 objMD.location : [{ key: objMD.location }];
+
+            if (dataLocator && dataLocator[0] &&
```

Comment:
add comment:
// if the data backend is azure, there will only ever be at most one item in the dataLocator array